integration_tests/suite/test_call_log_generation.py

```diff
@@ -1721,3 +1722,157 @@ def test_adhoc_conference(self):
+1674248467.40 | 1674248467.40 | 2023-01-20 21:01:23.491144+00 | ANSWER                    | PJSIP/rgcZLNGE-00000028 | 8000      | 98001      | test-UC-1  | 8000      | pickup            |                                                                                                                                                                                                                             
+1674248467.40 | 1674248473.42 | 2023-01-20 21:01:23.504964+00 | HANGUP                    | PJSIP/rNXlGVeY-0000002a | 8002      |            | B McTest   | 8002      | s                 | {"hangupcause":0,"hangupsource":"","dialstatus":""}                                                                                                                                                                         
+1674248467.40 | 1674248473.42 | 2023-01-20 21:01:23.504964+00 | CHAN_END                  | PJSIP/rNXlGVeY-0000002a | 8002      |            | B McTest   | 8002      | s                 |                                                                                                                                                                                                                             
+1674248467.40 | 1674248467.40 | 2023-01-20 21:01:29.900072+00 | HANGUP                    | PJSIP/rgcZLNGE-00000028 | 8000      | 98001      | test-UC-1  | 8000      | unreachable       | {"hangupcause":16,"hangupsource":"dialplan/builtin","dialstatus":"NOANSWER"}                                                                                                                                                
+1674248467.40 | 1674248467.40 | 2023-01-20 21:01:29.900072+00 | CHAN_END                  | PJSIP/rgcZLNGE-00000028 | 8000      | 98001      | test-UC-1  | 8000      | unreachable       |                                                                                                                                                                                                                             
+1674248467.40 | 1674248467.40 | 2023-01-20 21:01:29.900072+00 | LINKEDID_END              | PJSIP/rgcZLNGE-00000028 | 8000      | 98001      | test-UC-1  | 8000      | unreachable       |                                                                                                                                                                                                                             
+        '''
+    )
+    def test_user_missed_call_fallback_missed(self):
+        # user A missed, fallback to user B, missed too.
+        # Both were called, call log should include both as participants.
+        USER_A_UUID = "ad5b78cf-6e15-45c7-9ef3-bec36e07e8d6"
+        USER_B_UUID = "31be0853-dde6-48cd-986d-85bc708754a1"
+        tenant = "54eb71f8-1f4b-4ae4-8730-638062fbe521"
```

Comment:
Uppercase or lowercase for test constants, I don't care, but please keep consistent

Reply:
Also, why not using `USERS_TENANT`? I see that you changed it but I don't understand why.

Reply:
Because the tenant had to be consistent with the data in the CEL table of the test.